neovim/ui/gtk_ui.py

```diff
@@ -152,7 +152,11 @@ def _nvim_resize(self, columns, rows):
         self._cell_pixel_width = cell_pixel_width
         self._cell_pixel_height = cell_pixel_height
         self._screen = Screen(columns, rows)
-        self._window.resize(pixel_width, pixel_height)
+        prev_rect = self._drawing_area.get_allocation()
```

Comment:
@bfredl I think that is a good idea to fork the gtkui outside python-client. I can look at this and try to have some simple example.

I was also thinking about the resize problem as it is right now and there is something I do not understand about the **_nvim_resize** callback. It seams to me that it is expecting nvim to proactively decide to resize the screen without it being triggered by the user. Is there any case that it can happen? If that never happens we do not need to listen for this callback.